addons/api/addon/models/worker.js

```diff
@@ -34,14 +33,18 @@ export default class WorkerModel extends GeneratedWorkerModel {
   }
 
   /**
-   * Returns the number of canonical tags present on the worker.
+   * Returns the number of tags present on the worker.
    * @type {number}
    */
   get tagCount() {
-    if (!this.canonical_tags) {
+    if (!this.config_tags && !this.api_tags) {
       return 0;
     }
```

Comment:
I don't believe this if check is actually necessary since you do a default on lines 44 and 45 which should work fine and just add up two empty arrays to 0 anyways. 

```suggestion
```